Dockerfile

```diff
@@ -434,6 +434,10 @@ COPY openfeature-provider/java/src ./src/
 # Copy WASM module into resources
 COPY --from=wasm-rust-guest.artifact /confidence_resolver.wasm ../../../wasm/confidence_resolver.wasm
 
+
+# Copy WASM module into resources
+COPY openfeature-provider/java/.embedded-wasm-version ./.embedded-wasm-version
```

Comment:
We noticed that the HASH of the coped wasm from CI (Docker) is different than the hash we computed locally via Makefile (currently part of this PR's diffs.). We need to investigate